regulations/download-files.py

```diff
@@ -0,0 +1,124 @@
+"""Download Regulations.gov documents from indexes containing document URLs"""
+
+import argparse
+import itertools
+import json
+import os
+
+import requests
+from tqdm.auto import tqdm
+
+from licensed_pile import logs
+
+
+def parse_args():
+    parser = argparse.ArgumentParser("Regulations.gov index builder")
+    parser.add_argument(
+        "--input-dir", required=True, help="Path to directory containing indexes"
+    )
+    parser.add_argument("--output-dir", required=True, help="Path to output directory")
+    parser.add_argument(
+        "--years",
+        nargs="+",
+        default=list(map(str, range(2000, 2024))),
+        help="Years of data to process",
+    )
+    parser.add_argument(
+        "--agencies",
+        nargs="+",
+        default=[
```

Comment:
A read-only value like this should probably be a tuple instead of a list. There are a few other places that should have this same change